### PR TITLE
feat: Add Issues Counter to Issue Tab Title

### LINKED ISSUE #13: Add Open Issues Counter based on Filter
image

I'd like to propose adding a counter for open issues in the project, specifically reflecting the current filter settings. Here are some ideas for implementation:

**Display Options:**

- We could show the exact number of open issues.
- Alternatively, if the count exceeds 99, we could display "99+" for a cleaner look. (I'm not sure about it)

**Badge Style:**

- The counter could be styled as a badge for better visibility and aesthetics.
image

**What do you think of this idea? I'm happy to work on it further if there's interest!**

### PR DESCRIPTION
**Pull request by rog404**
#### Full Screen View:
image

#### Highlight of the Change:
image

#### Badge Not Displayed If No Issues Exist:
image

---

Close #13

### CODE REVIEW

File: components/ui/badge.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+interface BadgeProps {
+  children: React.ReactNode;
+  counter: number;
+}
+
+const Badge: React.FC<BadgeProps> = ({ children, counter }) => {
+  const badgeCounter = counter > 99 ? "99+" : counter;
```
Comment (ntheanh201) [inline components/ui/badge.tsx:9]: I don't think it's good if we tailored the Badge for only counter, it should be more generic use case for showing Badge content

Comment (swve) [reply components/ui/badge.tsx:9]: @ntheanh201 what additional content you're suggesting for example ?

Comment (ntheanh201) [reply components/ui/badge.tsx:9]: @swve for me, change the counter to `content`, the condition should move out of badge component, and pass it through props like this:
`<Badge content={issues.length > 99 ? "99+" : issues.length}>`

Comment (swve) [reply components/ui/badge.tsx:9]: I agree, we can pass tailwind colours too.

`<Badge tw-color="bg-red-600" content={issues.length > 99 ? "99+" : issues.length}>`

Comment (rog404) [reply components/ui/badge.tsx]: Thanks for the suggestions!

@swve 
I made a small change to the idea of `tw-color`. I added a `className` prop, which allows you to add both additional badge colors and new styles.

The default red remains unchanged.

Comment (ntheanh201) [reply components/ui/badge.tsx]: LGTM

Comment (swve) [reply components/ui/badge.tsx]: LGTM2

**Comment by swve** [comment]
Thanks @rog404 for your contribution 🤗 and @ntheanh201 for your reviews 🙏

**Pull request merged**